website/versioned_docs/version-0.63/cli.md

```diff
@@ -0,0 +1,77 @@
+---
+id: version-0.63-cli
+title: React Native Windows CLI
+original_id: cli
+---
+
+This guide will give you more information on the React Native Windows CLI.
+
+## `react-native-windows-init`
+
+`react-native-windows-init` is a CLI used to bootstrap the addition of the Windows platform to `react-native` projects.
+
```

Comment:
the challenge here is the init CLI is evergreen (there is only one), and the build CLI is per-version. So if we change the init cli, now we have to go update N copies of cli.md on the website.

Can this file be split into a init CLI and a per-version-build CLI file, and the init CLI doesn't go into versioned docs?
I mentioned to @jonthysell we should probably have some docs that don't get replicated for every version, he had some thoughts here too

Reply:
Yea that definitely makes sense. I'm not sure if we fully figured out yet how to do static pages yet, but I can look into it! Currently we have the init CLI options written out on every getting-started page for every version, so we've already dug ourselves a bit of a whole with the N copies but it definitely seems like the best path would be to do the investment in setting up static pges.

Reply:
Yeah, static pages are located in the website/pages folder. Unfortunately you have to write them as like actual ReactJS pages (there are examples in there), but I think it makes sense for the init command. Then we can just link to it from getting started, and from init's readme.

The full doc for `run-windows` and it's arguments should be versioned, and maybe worth renaming the doc from the generic cli.md to something like cli-run-windows.md.

Reply:
Sounds good!